docs/docusaurus/docs/core/configure_project_settings/configure_metadata_stores/configure_metadata_stores.md

```diff
@@ -0,0 +1,118 @@
+
+   GX Cloud accounts manage Stores for you online, and do not support custom Store configurations.
+
+   Therefore, only File Data Contexts can have customized Store configurations.  This procedure assumes you have a File Data Context loaded as the variable `context`:
+
+   ```python title="Python"
+   import great_expectations as gx
+   
+   context = gx.get_context(mode="file")
+   ```
+
+2. Determine the Store to update.
+
+   GX utilizes 5 Stores for different types of data and metadata.  These Stores are the Expectations Store, Validation Definitions Store, Checkpoint Store, Validation Results Store, and the Suite Parameter Store.  All Stores can be accessed by passing a corresponding key to a Data Context's `variables.stores` attribute.  The following code shows how to print the configuration for each of these stores:
+
+   ```python title="Python"
+   print(context.variables.stores['expectations_store'])
+   print(context.variables.stores['validation_definition_store'])
+   print(context.variables.stores['checkpoint_store'])
+   print(context.variables.stores['suite_parameter_store'])
+   print(context.variables.stores['validation_results_store'])
+   ```
+   
+   When updating a Store configuration you will use the same key to access it from your Data Context's `variables.stores` attribute as was used to print it in the above example.
+   
+3. Update the `base_path` of the Store to change.
+
+   Each Store has a `store_backend` configuration that determines how and where the Store accesses and saves information.  To change the location of a Store, you will update the `store_backend` configuration's `base_directory` value.  For instance, to change the location of an Expectation Store, you would update the `expectation_store_directory` variable in the following code and execute it:
+
+   ```pyhton title="Python"
+   expectation_store_directory = "my_expectations_store/"
+   context.variables.stores['expectations_store']['store_backend']['base_directory'] = expectation_store_directory
+   ```
+
+   The path provided for the `base_directory` should be either be an absolute path, or a path relative to the File Data Context's `project_root_dir`.
+
+4. Save the File Data Context variables.
+
+   Once a Data Context's `variables` have been updated, the changes need to be saved to the Data Context's configuration file so that they will persist when the Data Context is loaded in the future.  This is done with:
+
+   ```python title="Python"
+   context.variables.save()
+   ```
+
+5. Re-initialize the File Data Context.
+
+   Because Store configurations are loaded when the Data Context is initialized, you will need to re-initialize your Data Context before your changes will take effect.  This is done by loading the Data Context again, exactly as when it was loaded the first time:
```

Comment:
I didn't know this was true. I will file a ticket so we don't have to do this.

Reply:
https://greatexpectations.atlassian.net/browse/V1-433

Reply:
I did some digging -- this might be a mistake on my part.  The notes I finally turned up state that when toggling analytics through `context.variables` you need to re-initialize your Data Context for it to take effect, not when you update Stores.

Reply:
I believe this will be true for this right now. We have a bug tracking toggling analytics but the underlying issue will cause the same thing behavior here.